TvMaterialCatalog/app/src/main/java/com/google/tv/material/catalog/HomeGrid.kt

```diff
@@ -27,7 +39,10 @@ fun HomeGrid() {
         ) {
             TvLazyVerticalGrid(
                 columns = TvGridCells.Fixed(4),
-                modifier = Modifier.padding(top = 12.dp),
+                modifier = Modifier
+                    .padding(top = 12.dp)
+                    .focusRequester(focusRequester)
+                    .focusRestorer(),
```

Comment:
Removing this gives me an exception:

```
 java.lang.IllegalStateException: 
    FocusRequester is not initialized. Here are some possible fixes:
 
    1. Remember the FocusRequester: val focusRequester = remember { FocusRequester() }
    2. Did you forget to add a Modifier.focusRequester() ?
    3. Are you attempting to request focus during composition? Focus requests should be made in
    response to some event. Eg Modifier.clickable { focusRequester.requestFocus() }
```

Reply:
The `focusRequester` modifier is necessary as the `request` focus method is called over the associated `FocusRequester` object.  Without the modifier, calling `requestFocus` method crashes the app.

Reply:
Actually, focusRestorer doesn't need any focusRequester modifier to work. If the app is crashing, it could be a bug. Most likely it would be because of the 3rd point: "attempting to request focus during composition".

Reply:
I just noticed that there is a Launched Effect which is requesting focus on the focus requester. If the focusRequester is not initialized, it will crash the app. I think the goal here is to request focus on the first element. In that case, the focusRequest should be assigned to the first item's modifier and not the parent grid.